src/commands/indexer/index.ts

```diff
@@ -1428,11 +1477,21 @@
       `Sending cross chain transaction job to DBJobManager ${contractAddress}`,
       tags,
     )
+
+    const query = gql`
+    query($contractAddress: String!, $tokenId: String!) {
+      nftByContractAddressAndTokenId(contractAddress: $contractAddress, tokenId: $tokenId) {
+        id
+        contractAddress
+        tokenId
+      }
+    }
+    `
```

Comment:
i dont think this method is being used. should this be kpet here?

Reply:
I'm not 100% I know what you're referring to. The `query` is passed along into the db job for the callback via `updateCrossChainTransactionCallback` which is where the the actual API call is made. Let me know if it was something else I'm missing though

Reply:
@alexanderattar ok my bad. I mean to put that comment on the function `async sendMutationRequest(options: PatchOptions, tags: (string | number)[]): Promise<void>` on line 1518. When i dont believe this method is used anywhere. I think that method was moved over to `apiService.sendMutationRequest`.